docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -262,16 +268,20 @@ documentation.
 Contribution checklist
 ======================
 
-Use this checklist to review a pull request. If you are reviewing a pull
-request that is not your own and it passes all the criteria below, please set
-the "Triage Stage" on the corresponding Trac ticket to "Ready for checkin".
+Use this checklist to review to review a pull request.
+If you are reviewing a contribution that would not be
+:ref:`considered trivial <trivial-change>`, first ensure it has an accepted ticket.
+If the pull request passes all the criteria below and is not your own,
+please set the "Triage Stage" on the corresponding Trac ticket to "Ready for checkin".
```

Comment:
If you want a new paragraph, it needs an empty line
We also need to wrap the docs at 80 characters 👍 
```suggestion
Use this checklist to review a pull request. If this contribution would not be
:ref:`considered trivial <trivial-change>`, first ensure it has an accepted
ticket before proceeding with the review.

If the pull request passes all the criteria below and is not your own, please
set the "Triage Stage" on the corresponding Trac ticket to "Ready for checkin".
```

----

**Note:** we could also arguably avoid this change as it says:
> Use this checklist to review a pull request. If you are reviewing a pull
request that is not your own and it passes all the criteria below

and we have added the requirement of the accepted ticket to the checklist.

However, I think in this case it's worth it because we are trying to suggest a volunteer doesn't invest too much effort in reviewing something Django may have no intention of accepting